pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.sourcegraph.plugins</groupId>
     <artifactId>sourcegraph-bitbucket</artifactId>
-    <version>2.1.0</version>
+    <version>3.0.0</version>
```

Comment:
decided this is worth a major version bump as we'll mark this version as "requires BB9+" in the store.